flexmeasures/api/v3_0/tests/test_sensor_schedules_fresh_db.py

```diff
@@ -493,17 +493,18 @@ def test_multiple_contracts(
     print(consumption_schedule)
     print(soc_schedule)
 
-    # Check for absence of consumption breaches over 1 kW, i.e. any breach costs are avoided
-    assert all(v <= 0.001 for v in consumption_schedule)
+    # Check for consumption breaches over 0.25 kW, i.e. any breach costs are avoided as much as possible
+    assert all(v <= 0.000298 for v in consumption_schedule)
```

Comment:
Where does this limit come from? When I computed the formula in line 418, I got `0.000274`

Reply:
Efficiencies. I'm now turning to using perfect efficiencies for this test, so the numbers come out exactly. I'll also describe better the steps of the calculation, as inline documentation.